kubectl-plugin/kubectl-waiops

```diff
@@ -454,6 +454,19 @@ getODLMPodStatus() {
     fi
 }
 
+getAllCertStatus() {
+    AIOPS_CERTS=()                                                                                                                                                                                                             
+    while IFS= read -r line; do
+        AIOPS_CERTS+=( "$line" )
+    done < <( oc get certificate -n $INSTALLATION_NAMESPACE -o=custom-columns='NAME:.metadata.name' --no-headers=true )
+    echo "Certificate Name               Renewal Time"
+    for c in "${AIOPS_CERTS[@]}"; do
+        certStatus=$(oc get cert $c --no-headers=true | awk '{print $2}')
+        renewalTime=$(oc get cert $c -o jsonpath='{.status.renewalTime}')
+        printStatus "$certStatus" "True" "$c    $renewalTime"
+    done
+}
+
```

Comment:
The output looks a bit messy with the white header for the cert section (see photo below). LIke how we do with the OperandRequest instances section, can we create some custom columns so that each line has its own `Certificate Name` and `Renewal Time` headers that match the color-coding? Also this way, the output will look neater.

<img width="726" alt="Screenshot 2024-01-30 at 9 15 07 AM" src="https://github.com/IBM/cp4waiops-samples/assets/91081503/e37ba4e1-8032-45a4-850b-4431a21c1421">

Reply:
I switched to custom columns, so the output is now (including a fake certificate to show a non-ready cert):

<img width="920" alt="Screenshot 2024-01-30 at 12 28 51 PM" src="https://github.com/IBM/cp4waiops-samples/assets/18495025/5d752989-f165-4199-add8-68f26bfc9ac6">

Reply:
Looks great. I think if we can remove [L462](https://github.com/IBM/cp4waiops-samples/pull/216/files#diff-2ebe69d8ea0e70e35ebfb7ba59e7e9ba7eaa6955acf43f19b6e0c43db2f132c5R462) (the white header at the top) we should be good to go.